### PR TITLE
Include PetitDetector

### PR DESCRIPTION
**Pull request by carmenromo**
A new class for the PETit detector has been included. The electric field has been set to 10 V/cm in the whole volume.

### CODE REVIEW

File: include/Detectors/PetitDetector.hh [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  // s1polA + s1polB*z[mm] + s1polC*z^2+... (QE included, for binom dist) e.g.
+  double FitS1(double /*xPos_mm*/, double /*yPos_mm*/, double /*zPos_mm*/,
+               LCE /*map*/) override {
+    return 1.;  // unitless, 1.000 at detector center
+  }
+
+  // Drift electric field as function of Z in mm
+  // For example, use a high-order poly spline
+  double FitEF ( double xPos_mm, double yPos_mm, double zPos_mm ) override { // in V/cm
+    double field = 1;
```
Comment (paolafer) [inline include/Detectors/PetitDetector.hh]: You can simplify with `return 1`.

File: include/Detectors/PetitDetector.hh [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    // Ionization and Secondary Scintillation (S2) parameters
+    g1_gas = .0655;  // phd per S2 photon in gas, used to get SE size
+    s2Fano = 3.61;   // Fano-like fudge factor for SE width
+    s2_thr = 300.;  // the S2 threshold in phe or PE, *not* phd. Affects NR most
+    E_gas = 12.;    // field in kV/cm between liquid/gas border and anode
+    eLife_us = 2200.;  // the drift electron mean lifetime in micro-seconds
+
+    // Thermodynamic Properties
+    inGas = false;
+    T_Kelvin = 163.; // T of the value of density used in G4
```
Comment (paolafer) [inline include/Detectors/PetitDetector.hh]: Let's use T = 164 and P = 1.26, which are the conditions of the real detector.

**Review by paolafer — CHANGES_REQUESTED** [review]

File: include/Detectors/PetitDetector.hh (hunk 2 above)
Comment (carmenromo) [reply include/Detectors/PetitDetector.hh]: You're right, I forgot!

**Review by paolafer — APPROVED** [review]
Simple PR, which adds the configuration for PETit. Approved!

**Pull request merged**